src/transformers/models/auto/tokenization_auto.py

```diff
@@ -771,6 +772,7 @@ def from_pretrained(cls, pretrained_model_name_or_path, *inputs, **kwargs):
         use_fast = kwargs.pop("use_fast", True)
         tokenizer_type = kwargs.pop("tokenizer_type", None)
         trust_remote_code = kwargs.pop("trust_remote_code", None)
+        from_gguf = kwargs.get("from_gguf", None)
```

Comment:
here we default to None vs defaulting to False

Reply:
It has been harmonized